docs/src/main/paradox/release-notes/10.1.x.md

```diff
@@ -8,6 +8,42 @@ ALPN support was backported to recent JDK 8 updates. When using HTTP2 support wi
 is not needed anymore. If you want to run on older and newer JDKs with the same command line, make sure to use the
 most recent version of `jetty-alpn-agent` which will automatically disable itself for newer JDKs.
 
+### Changes since 10.1.11
+
+#### akka-http-core
+
+* Fix EOL detection for body part parsing [#3144](https://github.com/akka/akka-http/pull/3144)
+* Fix parsing of partly cached headers with UTF-8 values [#3096](https://github.com/akka/akka-http/pull/3096)
+* Fix cancellation race conditions on the client-side [#2965](https://github.com/akka/akka-http/pull/2965)
+* Make sure to cancel response entity on failure [#3046](https://github.com/akka/akka-http/pull/3046)
+* Make sure reference.conf files end with a new-line character.[#2841](https://github.com/akka/akka-http/pull/2841)
+* Only catch NonFatal Exceptions while parsing responses [#2853](https://github.com/akka/akka-http/pull/2853)
+* Add the remote address in parse errors when possible (when remote header enabled) [#2899](https://github.com/akka/akka-http/pull/2899)
+* Silence outgoing request stream error (and make it more useful) [#2816](https://github.com/akka/akka-http/pull/2816)
+* Identify Content-Type `charset` parameter even if not lower case [#2926](https://github.com/akka/akka-http/pull/2926)
+* Prevent initialization NPE which might fail all retries quickly [#2958](https://github.com/akka/akka-http/pull/2958)
+* Add exclusion for Extension issues when building against Akka 2.6 [#2945](https://github.com/akka/akka-http/pull/2945)
+* Nest correctly in NewHostConnectionPool [#2964](https://github.com/akka/akka-http/pull/2964)
+
+#### akka-http-marshallers
+
+* Jackson: better JSON validation error when unmarshalling [#2901](https://github.com/akka/akka-http/pull/2901)
+
+#### docs
+
+* Small typo in docs/src/main/paradox/common/marshalling.md [#2864](https://github.com/akka/akka-http/pull/2864)
+* Add warning on usage on extractClientIP [#2922](https://github.com/akka/akka-http/pull/2922)
+* Show RequestBuilding in client examples [#2968](https://github.com/akka/akka-http/pull/2968)
+* Fix project-info links to API docs [#2857](https://github.com/akka/akka-http/pull/2857)
+
+#### akka-http2-support
+
+* Support ALPN natively on JDK >= 8u252 [#3125](https://github.com/akka/akka-http/pull/3125)
```

Comment:
As this is the main driver for releasing you should put a more prominent note directly under "Changes since".

Reply:
even better, it's between "10.1.12" and "Changes since" :)

Reply:
It's already there just above that ;)